lib/transformers/govukDateInput.js

```diff
@@ -91,12 +91,17 @@ module.exports = ({schemaKey, schema, options, data, schemaErrors} = {}) => {
         opts.macroOptions.fieldset.legend.classes = 'govuk-fieldset__legend--xl';
     }
 
-    // Get only the date parts that should be displayed
-    opts.macroOptions.items = opts.macroOptions.items.filter(item => {
-        const datePart = item.label.toLowerCase();
+    if (opts.macroOptions.autocomplete) {
+        opts.macroOptions.items[0].autocomplete = `${options.macroOptions.autocomplete}-day`;
+        opts.macroOptions.items[1].autocomplete = `${options.macroOptions.autocomplete}-month`;
+        opts.macroOptions.items[2].autocomplete = `${options.macroOptions.autocomplete}-year`;
+        delete opts.macroOptions.autocomplete;
```

Comment:
remove redundant property

Reply:
Is it necessary to delete this?

Reply:
This property has served its purpose of enabling the items to be populated with the autocomplete attribute.. It is a property that is not needed to be passed to the nunjucks template in this instance (date input). So I removed it to avoid confusion and reduce clutter.